test/test_commands_serve.rb

```diff
@@ -262,10 +262,10 @@ def serve(opts)
         should "not update the site url" do
           expect(Jekyll).to receive(:env).and_return("production")
           expect(Jekyll::Commands::Serve).to receive(:start_up_webrick)
-          @merc.execute(:serve, { "watch" => false, "url" => "https://jekyllrb.com/" })
+          @merc.execute(:serve, { "watch" => false, "url" => "https://jekyllrb.com" })
```

Comment:
This will probably break many sites in weird ways. We need to document this really well.